ports/servo/build.rs

```diff
@@ -83,3 +89,10 @@ fn android_main() {
     println!("cargo:rustc-link-lib=log");
     println!("cargo:rustc-link-lib=android");
 }
+
+fn windows_main() {
+    let mut res = winres::WindowsResource::new();
+    res.set_resource_file("windows.rc");
```

Comment:
@UK992 Thanks for the suggestion, it would simplify the code a lot! :) However, only a corrupted version of the icon shows up in my taskbar... 

![servoicon](https://cloud.githubusercontent.com/assets/7758155/24889110/278a3b4e-1e36-11e7-8d3f-ca99e21d56d9.PNG)

Do you think this is a problem with `winres`?